### PR TITLE
Local and Developing Locally docs: update intros

### LINKED ISSUE #1113: Local Doc Update
Re: [Local](https://pantheon.io/docs/articles/local)
## Issue Description:
Duplicate intro in [Local](https://pantheon.io/docs/articles/local) landing page and in [Developing Locally](https://pantheon.io/docs/articles/local/developing-locally/)
## Suggested Resolution 
Remove or revise statement to avoid duplicate content.

### PR DESCRIPTION
**Pull request by nataliejeremy**
Closes #1113 

@bmackinney or @rachelwhitton please review.

### CODE REVIEW

File: source/docs/articles/local.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ category:
 - getting-started
 keywords: local development, pantheon, develop locally, developing
 ---
-While Pantheon provides several options for on-server development, [local development](/docs/articles/local/developing-locally) has a number of advantages, especially if continuous Internet access is a concern. Pantheon cannot troubleshoot or support local development solutions; however, we can provide some suggestions and known working solutions. If you're looking for a self-contained local development solution, check out [Kalabox](http://www.kalamuna.com/products/kalabox), which integrates with the Pantheon platform.
+While Pantheon provides several options for on-server development, [local development](/docs/articles/local/developing-locally) has a number of advantages, especially if continuous Internet access is a concern. Pantheon cannot troubleshoot or support local development solutions; however, we can provide some suggestions and known working solutions.
```
Comment (rachelwhitton) [inline source/docs/articles/local.md]: This still feels like leftover content from the local setup doc, is anyone opposed to deleting the intro entirely here? The existing header, **Resources for Local Development**, is good enough for describing the subject of the listed docs IMO

File: source/docs/articles/local/developing-locally.md [hunk 2]
```diff
@@ -5,7 +5,8 @@ category:
 - getting-started
 keywords: local development, pantheon, develop locally, developing
 ---
-While Pantheon provides several options for on-server development, local development has a number of advantages, especially if continuous Internet access is a concern. Pantheon cannot troubleshoot or support local development solutions, however we can provide some suggestions and known working solutions.
+Local development has a number of advantages, especially if continuous Internet access is a concern. This article will walk you through setting up a local development environment.
```
Comment (rachelwhitton) [inline source/docs/articles/local/developing-locally.md]: Suggest restoring original intro within this doc, given that the duplicate intro in the local.md landing page is deleted. The forewarning about Pantheon's inability to troubleshoot local developments should stay

**Comment by nataliejeremy** [comment]
Thanks @rachelwhitton

**Comment by rachelwhitton** [comment]
+1 looks good, thanks @nataliejeremy!

**Pull request merged**